extensions/bitwarden/src/components/searchVault/actions/CopyPublicKeyAction.tsx

```diff
@@ -0,0 +1,41 @@
+import { Clipboard, Icon, showToast, Toast } from "@raycast/api";
+import ActionWithReprompt from "~/components/actions/ActionWithReprompt";
+import { useSelectedVaultItem } from "~/components/searchVault/context/vaultItem";
+import useGetUpdatedVaultItem from "~/components/searchVault/utils/useGetUpdatedVaultItem";
+import { showCopySuccessMessage } from "~/utils/clipboard";
+import { captureException } from "~/utils/development";
+
+function CopyPublicKeyAction() {
+  const selectedItem = useSelectedVaultItem();
+  const getUpdatedVaultItem = useGetUpdatedVaultItem();
+
+  if (!selectedItem.sshKey?.publicKey) return null;
+
+  const handleCopyPublicKey = async () => {
+    try {
+      const publicKey = await getUpdatedVaultItem(
+        selectedItem,
+        (item) => item.sshKey?.publicKey,
+        "Getting public key..."
+      );
+      if (publicKey) {
+        await Clipboard.copy(publicKey);
```

Comment:
We should pass the transient flag depending on the preference here too.

```suggestion
        await Clipboard.copy(publicKey, { transient: getTransientCopyPreference("other") });
```